ubuntu/Dockerfile

```diff
@@ -1,16 +1,23 @@
 FROM ubuntu:xenial
 LABEL maintainer="Kong Core Team <team-core@konghq.com>"
 
-ENV KONG_VERSION 2.0.0
+# VERSION='2.0.0' community
```

Comment:
shouldn't this one also use a "downloader" to be on a par with the other ones?

Also; isn't there a way to remove the repeated downloader config out and de-duplicate it?

Reply:
There is a way to de-duplicate but it wouldn't be compatible with docker's automated build system.

Reply:
ok, but then still shouldn't this one use a downloader for consistency? so with future changes we can simply replicate the same change in every platform-file?